lib/hyperion/result_handling/result_maker.rb

```diff
@@ -35,6 +36,8 @@ def make_from_typho(typho_result, result_class)
       read_body = proc { read(typho_result.body, route.response_descriptor) }
       code = typho_result.code
 
+      log_error_response(read(typho_result.body, :json)) unless typho_result.success?
```

Comment:
What difference does it make that this is "private"? How is someone supposed to confidently and easily refactor this class without a deep understanding of all of Hyperion?